app/assets/stylesheets/global/_vars.scss

```diff
@@ -14,6 +14,7 @@ $font_weight-bold: 700;
 $color_bg-lightest: #FFFFFF;
 $color_bg-light: #EFEFE7;
 $color_bg-dark: #006e47;
+$color_bg-black: #000000;
```

Comment:
Why go with a variable rather than directly referencing `black`?

Reply:
We'll probably never redefine black, but it makes it a little clearer to search for if we're ever looking for where black is used. It also keeps the consistency of always using variables for colors.